portfolio/src/main/webapp/script.js

```diff
@@ -28,15 +38,45 @@ function addRandomGreeting() {
 }
 
 function showPicture() {
```

Comment:
please add docs for each function

Reply:
Updated, let me know if my docs aren't up to style when I commit.